src/main/java/flight/reservation/plane/Plane.java

```diff
@@ -0,0 +1,77 @@
+package flight.reservation.plane;
+
+public abstract class Plane {
+    protected String model;
+    protected int passengerCapacity;
```

Comment:
Arguable design choice. Subclasses have to be aware that these variables have to be initialized. Subclasses should be forced to provide a value by a corresponding constructor or abstract method etc.